.changeset/purple-turtles-design.md

```diff
@@ -0,0 +1,5 @@
+---
+"ggt": minor
```

Comment:
```suggestion
"ggt": patch
```

This should be considered a `patch` rather than a `minor` change, since ggt isn't v1 yet.

```
// breaking change
minor = v0.4.8 -> v0.5.0

// non-breaking change
patch = v0.4.8 -> v0.4.9
```